pageserver/src/http/routes.rs

```diff
@@ -267,13 +320,13 @@ async fn tenant_create_handler(mut request: Request<Body>) -> Result<Response<Bo
 
     let request_data: TenantCreateRequest = json_request(&mut request).await?;
 
-    let response_data = tokio::task::spawn_blocking(move || {
+    tokio::task::spawn_blocking(move || {
         let _enter = info_span!("tenant_create", tenant = %request_data.tenant_id).entered();
```

Comment:
Not your issue, but the span name is definitely odd :)
```suggestion
        let _enter = info_span!("tenant_list_handler", tenant = %request_data.tenant_id).entered();
```

Reply:
This is actually tenant create handler, I was just passing by and removed odd `response_data` represented by unit type. Diff visualization is indeed confusing